tests/app/csv_test.py

```diff
@@ -0,0 +1,28 @@
+"""
+Tests for CSV generation code (CSV endpoints are tested in ``public_test.py``
```

Comment:
actually - maybe also move the CSV endpoint tests here too, since this is where the CSV endpoints are?

Reply:
Sure. Everything's in csv_test.py now.